scripts/fullstack/README.md

```diff
@@ -0,0 +1,181 @@
+   ```
+
+   You should enter the following after you have installed the working
+   `findy-agent-cli`:
+   ```shell
+   export FCLI=<your-name-for-binary>
+   ```
+   That's for the helper scrips used in this directory and referenced here as
+   well.
+
+   **Admin Operations**
+
+   After environment setup you can see what your configuration is by executing
+   the following helper script:
+   ```shell
+   admin/cli-env
+   ```
+   It will output all of the `findy-agent-cli` env configurations currently set.
+   To check one specific variable enter: `admin/cli-env KEY` for example.
+
+   To register your CLI authenticator for direct communication to Findy Agency
+   enter the following commands:
+   ```shell
+   source admin/register
+   source admin/login
+   ```
+   Later the login is all what is needed. After successful login you can enter
+   commands like:
+   ```shell 
+   $FCLI agency count         # get status of the clould agents
+   $FCLI agency logging -L=5  # set login level of the core agency 
+   ```
+
+   **On-board Alice and Bob**
+   ```shell
+   source alice/register
+   source bob/register
+   ```
+   You can play each of them by entering for example following:
+   ```shell
+   source alice/login
+   $FCLI agent ping
+   ```
+
+   **Alice invites Bob to connect**
+
+   ```shell
+   export FCLI_CONN_ID=`alice/invitation | bob/connect`
+   ```
+   Or on macOS could be convenient to have it in clipboard as well:
+   ```shell
+   alice/invitation | bob/connect | pbcopy && export FCLI_CONN_ID=pbpaste
+   ```
+
+   Now you have the connection ID (pairwise ID) in the environment variable and
+   you could test that with the commands:
+   ```shell
+   source alice/login
+   $FCLI connection trustping
+   ```
+   Which means that Alice's end of the connection calls Aries's trustping
+   protocol and Bob's cloud agent responses it.
+
+   Before entering previous commands you could open a second terminal window and
+   execute following:
+   ```shell
+   source ./use-key.sh
+   source ./setup-cli-env.sh
+   source bob/login
+   export FClI_CONN_ID="<perviously defined conn id here>"
+   $FCLI agent listen
+   ```
+   You should now receive a notification of the trustping protocol.
+
+   **Alice sends text message to Bob**
+   First in the Bob's terminal stop the previous listening with C-c and enter
+   the following:
+   ```shell
+   $FCLI bot read
+   ```
+   Go to the Alice's terminal and enter the commands:
+   ```shell
+   source alice/login
+   echo "Hello Bob! Alice here." | $FCLI bot chat
+   ```
+   The Bob's terminal should output Alice's welcoming messages. To stop Bob's
+   listen command just press C-c.
+
+   **TODO: What else we really need for now?**
+
+   Should we have issuing example, chat bots, all of them or should we wait for
+   the feedback first and bring them if there is demand.
+
+1. [Optional, requires Node.js] Start wallet development environment. On the
```

Comment:
Remove this or add pointers to PWA repository?

Reply:
Thanks 👍